.gitignore

```diff
@@ -6,3 +6,4 @@ coverage/
 .cache
 dist/
 node-warnings.logs
+.idea/
```

Comment:
You need to put it to your local git ignore

https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Okay. Sorry about it.